dbtwiz/main.py

```diff
@@ -20,7 +21,10 @@ class InvalidArgumentsError(ValueError):
 
 
 # Add admin commands as subcommands of 'admin'
-app.add_typer(admin.app, name="admin")
+app.add_typer(admin.app, name="admin", help="Administrative commands")
+
+# Add generator commands as subcommands of 'generate'
```

Comment:
Is `generate` a good command name? Or would it be simpler and easier to understand if it was `create`?

Reply:
I'm not sure what most clearly describes what it does. I picked `generate` mostly because I'm environmentally damaged by years of Rails use. :)

Reply:
Renamed to `create`